libraries/oneview_helper.rb

```diff
@@ -55,10 +60,12 @@ def self.load_sdk(context)
     rescue LoadError => e
       Chef::Log.debug("Could not load gem oneview-sdk #{node['oneview']['ruby_sdk_version']}. Message: #{e.message}")
       Chef::Log.info("Could not load gem oneview-sdk #{node['oneview']['ruby_sdk_version']}. Making sure it's installed...")
+      # :nocov:
```

Comment:
the nocov is here because I checked the specs and, it's clearly passing through here, but simplecov cannot detect it executed the block inside the chef_gem, although being confirmed it did.

tl;dr Simplecov cannot detect this thing inside being executed, but it's being covered.